hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationEndpoint.java

```diff
@@ -497,6 +497,33 @@ public boolean canReplicateToSameCluster() {
     }
   }
 
+  public static class SleepingReplicationEndpointForTest extends ReplicationEndpointForTest {
```

Comment:
AFAICT this only gets used in manual testing? add a comment on doing that. something like your example for this PR.

Reply:
Will do.